recipe/meta.yaml

```diff
@@ -1,25 +1,27 @@
 {% set name = "pycosat" %}
-{% set version = "0.6.3" %}
+{% set version = "0.6.4" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  git_url: https://github.com/mingwandroid/pycosat.git
-  git_tag: {{ version }}
+  url: https://github.com/conda/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 79ffa99156810e58876dd6374af0b7bff88f9a4580ca8b8468c414c5c2454bb8
 
 build:
-  number: 1
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
```

Comment:
We do not provide cross compiling so these two lines can be removed

Reply:
would it be OK to stay more equal to the conda-forge recipe

Reply:
It can be but we removed them for our feedstocks